api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -917,4 +924,35 @@ public void testUnableToShareWithNonExistentUser() throws Exception {
     shareWorkspaceRequest.addItemsItem(writer);
     workspacesController.shareWorkspace(workspace.getNamespace(), workspace.getName(), shareWorkspaceRequest);
   }
+
+  @Test
+  public void testNoteBookList() throws Exception {
+    org.pmiops.workbench.firecloud.model.WorkspaceResponse fcResponse =
+        new org.pmiops.workbench.firecloud.model.WorkspaceResponse();
+    org.pmiops.workbench.firecloud.model.Workspace mockWorkspace = new org.pmiops.workbench.firecloud.model.Workspace();
+    mockWorkspace.setBucketName("MockBucketName");
+    fcResponse.setWorkspace(mockWorkspace);
+    when(fireCloudService.getWorkspace("mockProjectName", "mockWorkspaceName")).thenReturn(
+        fcResponse
+    );
+    List<String> blobDetailList = new ArrayList<String>();
```

Comment:
opt: I would probably just write:

`List<String> blobDetailList = ImmutableList.of("...", "...", "...")`

Alternatively, if you keep the ArrayList it can just be `new ArrayList<>();`